dio/lib/src/dio.dart

```diff
@@ -208,13 +208,13 @@ abstract class Dio {
   ///
   /// [lengthHeader] : The real size of original file (not compressed).
   /// When file is compressed:
-  /// 1. If this value is 'content-length', the `total` argument of `onProgress` will be -1
+  /// 1. If this value is 'content-length', the `total` argument of `onReceiveProgress` will be -1
```

Comment:
If you refer to the parameters via code references, it should also survive refactorings. Do you mind changing it for all occurrences?

```suggestion
  /// 1. If this value is 'content-length', the `total` argument of [onReceiveProgress] will be -1
```

Reply:
I've just updated the PR with the requested changes and some others I could find

Thanks!